### ISSUE TITLE
[Installation] Mac users with M1 processor fail to start demo-pokemon-api in docker compose

### ISSUE THREAD

**Issue by kretel** [kubeshop/tracetest#1684]
**Issue while installing Tracetest using docker-compose on OSX machines with M1 processor**

After installation of Tracetest you can't run full Tracetest instance including the demo-pokemon-api as the docker-compose points to latest image of pokemon-api that is only built for amd64 - [demo-pokemon-api/tags](https://hub.docker.com/r/kubeshop/demo-pokemon-api/tags)

**To Reproduce**
1. Install Tracetest following instructions from the documentation 'Using Docker Compose': 
``` 
brew install kubeshop/tracetest/tracetest
tracetest server install
```
2. Try to start your containers with command: 
`docker compose -f docker-compose.yaml -f tracetest/docker-compose.yaml up -d`
4. Notice that some services won't start: 
```
tracetest-demo-worker-1
tracetest-demo-api-1
tracetest-demo-rpc-1
```
3. You can still see the Tracetest in action but you can't use the `demo-pokemon-api` for creating any tests. 

**Work around**
1. Clone the [pokeshop](https://github.com/kubeshop/pokeshop) repository
`git clone git@github.com:kubeshop/pokeshop.git`
3. Build container: 
`docker build . --file Dockerfile --tag tmp/demo-pokemon-api:latest-arm64`
4. Replace values in `tracetest/docker-compose.yml' file from previous installation of the Tracetest:
`kubeshop/demo-pokemon-api:latest` should be swapped to `tmp/demo-pokemon-api:latest-arm64`
5. Start your Tracetest with command: 
`docker compose -f docker-compose.yaml -f tracetest/docker-compose.yaml up -d`

**Additional context**
1. It will be nice to have a choice during installation between Intel and arm64 processor. 
2. More user friendly approach is to automatically detect user's machine architecture and build the correct `docker-compose.yaml` file. 
I feel that the latter option is smoother and easier to implement.

**Comment by kretel** [comment]
@schoren , thanks for clarification. You're correct, when I reinstalled and started fresh it seems that all works as expected: 

<img width="1528" alt="image" src="https://user-images.githubusercontent.com/3034222/207651926-a1e4f115-8189-454f-8946-9bd905654169.png">

Sounds like not an issue as the application works as expected. However, we may want to remove the warning. 
Should I close it or you can close the issue?

**Comment by kdhamric** [comment]
@xoscar @schoren Any thoughts on if this warning can be silenced without extensive effort?

**Comment by schoren** [comment]
I'm trying to setup multiarch build. I forked the pokehsop repo and I'll run some tests there. It shouldn't bee too difficult. If that works, I'll also implement it on the main tracetest repo. I'll get back here with updateds

**Comment by schoren** [comment]
It's already implemented on [pokeshop](https://github.com/kubeshop/pokeshop/commit/0a161f026064faa4b6350f6f1b24cbcf3142a5d6) and there's a PR to implement it on tracetest: https://github.com/kubeshop/tracetest/pull/1711

**Comment by schoren** [comment]
@kdhamric @kretel It's been fixed. The image is now multi arch, so you shouldn't see the warning anymore.

This is only in the `latest` tag, so the last released version (v0.8.2) will still show the warning. The next release should come with the fix included
